server/src/main/kotlin/org/javacs/kt/SourcePath.kt

```diff
@@ -17,7 +18,7 @@ class SourcePath(
     private inner class SourceFile(
             val uri: URI,
             var content: String,
-            val path: Path? = null,
+            val path: Path? = try { Paths.get(uri) } catch (e: InvalidPathException) { null },
```

Comment:
Consider using `runCatching { Paths.get("") }.getOrNull()`

Reply:
Thanks for the suggestion. I am a little unsure about whether we should give up on the specific exception, though I assume that should not be a problem since `InvalidPathException` is the only thrown exception that `Paths.get` declares.

Reply:
Implemented in 4ca312bf09273a2a26feb094482de26a0a233ddc